system/Pager/PagerRenderer.php

```diff
@@ -117,7 +117,7 @@ public function getPrevious()
 
 		$uri = clone $this->uri;
 
-		$uri->addQuery('page', $this->first - 1);
+		$uri->addQuery('page', $this->current - 1);
```

Comment:
Yes, the code is the same as the documentation. But this logic is not good. The vast majority of paging is not the case.
There seems to be no example of handling paging.